src/main/java/overlap_crap_fix_locations_later/frameworks_and_drivers/OverlapInputDialog.form

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Make sure that the Markdown file notes the prerequisites to compile this IntelliJ UI.

Reply:
I added the reqs to the README since I couldn't open the XML Properly in IntelliJ.